Cargo.toml

```diff
@@ -158,29 +161,6 @@ mcu-STM32L051R8Hx = ["io-STM32L051", "stm32l0x1", "tfbga64"]
 mcu-STM32L051R8Tx = ["io-STM32L051", "stm32l0x1", "lqfp64"]
 mcu-STM32L051T6Yx = ["io-STM32L051", "stm32l0x1", "wlcsp36"]
 mcu-STM32L051T8Yx = ["io-STM32L051", "stm32l0x1", "wlcsp36"]
-mcu-STM32L052C6Tx = ["io-STM32L051", "stm32l0x2", "lqfp48"]
-mcu-STM32L052C8Tx = ["io-STM32L051", "stm32l0x2", "lqfp48"]
-mcu-STM32L052K6Tx = ["io-STM32L051", "stm32l0x2", "lqfp32"]
-mcu-STM32L052K6Ux = ["io-STM32L051", "stm32l0x2", "ufqfpn32"]
-mcu-STM32L052K8Tx = ["io-STM32L051", "stm32l0x2", "lqfp32"]
-mcu-STM32L052K8Ux = ["io-STM32L051", "stm32l0x2", "ufqfpn32"]
-mcu-STM32L052R6Hx = ["io-STM32L051", "stm32l0x2", "tfbga64"]
-mcu-STM32L052R6Tx = ["io-STM32L051", "stm32l0x2", "lqfp64"]
-mcu-STM32L052R8Hx = ["io-STM32L051", "stm32l0x2", "tfbga64"]
-mcu-STM32L052R8Tx = ["io-STM32L051", "stm32l0x2", "lqfp64"]
-mcu-STM32L052T6Yx = ["io-STM32L051", "stm32l0x2", "wlcsp36"]
-mcu-STM32L052T8Fx = ["io-STM32L051", "stm32l0x2", "wlcsp36"]
-mcu-STM32L052T8Yx = ["io-STM32L051", "stm32l0x2", "wlcsp36"]
-mcu-STM32L053C6Tx = ["io-STM32L051", "stm32l0x3", "lqfp48"]
-mcu-STM32L053C8Tx = ["io-STM32L051", "stm32l0x3", "lqfp48"]
-mcu-STM32L053R6Hx = ["io-STM32L051", "stm32l0x3", "tfbga64"]
-mcu-STM32L053R6Tx = ["io-STM32L051", "stm32l0x3", "lqfp64"]
-mcu-STM32L053R8Hx = ["io-STM32L051", "stm32l0x3", "tfbga64"]
-mcu-STM32L053R8Tx = ["io-STM32L051", "stm32l0x3", "lqfp64"]
-mcu-STM32L062K8Tx = ["io-STM32L051", "stm32l0x2", "lqfp32"]
-mcu-STM32L062K8Ux = ["io-STM32L051", "stm32l0x2", "ufqfpn32"]
-mcu-STM32L063C8Tx = ["io-STM32L051", "stm32l0x3", "lqfp48"]
-mcu-STM32L063R8Tx = ["io-STM32L051", "stm32l0x3", "lqfp64"]
```

Comment:
Regarding the re-grouping: Note that this code is generated (with https://github.com/dbrgn/cube-parse/), so I'd suggest not to group them by hand (or alternatively, extend the generator to support these comments).